src/main/java/Control/Connect/DbProvider.java

```diff
@@ -124,7 +124,7 @@ private void afterSqlExecution(boolean successful) {
                 CustomLogs.Error("Set autocommit true failed." + e.getMessage());
             }
 
-            //disconnect(); - probably will not be needed here
+            disconnect(); //- probably will not be needed here
```

Comment:
cize uz to je potrebne?

Reply:
hej, to je zjavne potrebne aby sa vracali veci do connection poolu - skúšam o dva dni uvidíme po bežani na serveri

Reply:
ak to pojde, mozes odstranit ten komentar :D